worlds/shivers/Options.py

```diff
@@ -107,6 +124,61 @@ class PuzzleCollectBehavior(Choice):
     default = 1
 
 
+# Need to override the default options to remove the goal items and goal locations so that they do not show on web.
```

Comment:
This whole process doesn't really fix everything. Plando still exists, among other things, so it's still going to run into problems, it's also kind of "hacky" to overwrite all of these. Can you really not do anything else?

Reply:
I mean this is what was suggested, if there's a better way I would love to know.

Reply:
Just using an event?

Reply:
As far as I can tell, the only thing this is being used for is the completion_condition, so I'm not sure why this isn't just an event (ID None)

Reply:
I think I mentioned it before but I was trying to be weird and have an in joke goal item that displays when you beat the game. Event items will not do that.
 
However something like the Map Complete items in the Dooms/Heretic would be similar and display in these options when they probably should not. afaik this is the only game so far attempting do this, but nothing says it can't. Plando is of course still a problem, but that requires deeper knowledge, the web based options is at least something that can be mitagated.

Reply:
Can't you just make the client display the item anyway? Just fake one, basically?

Reply:
Idk, I'll still leave my approval, but I'm starting to like less and less all the weird things worlds are doing with non-event "events" XD

Reply:
At least this cleans up all the other stuff that should actually have been events but weren't and added events to clean up rule logic. I just want my one small weird thing lol.